packages/js/src/ui/components/Inbox.tsx

```diff
@@ -73,7 +74,15 @@ export const InboxContent = (props: InboxContentProps) => {
   });
 
   return (
-    <div class={style('inboxContent', 'nt-h-full nt-flex nt-flex-col')}>
+    <div
+      class={style(
+        'inboxContent',
+        cn('nt-h-full nt-flex nt-flex-col [&_.nv-preferencesContainer]:nt-pb-8 [&_.nv-notificationList]:nt-pb-8', {
+          '[&_.nv-preferencesContainer]:nt-pb-8 [&_.nv-notificationList]:nt-pb-8': isDevelopmentMode(),
+          '[&_.nv-preferencesContainer]:nt-pb-4 [&_.nv-notificationList]:nt-pb-4': !isDevelopmentMode(),
+        })
```

Comment:
this is the fix for the footer issue in dev mode that is on top of the notifications/preferences list; it was not allowing to click the items that were at the bottom of the list

Reply:
There was a bug about it, great!

Reply:
I found it and updated: https://linear.app/novu/issue/NV-5734/inbox-updating-the-last-workflow-preferences-in-dev-mode-impossible